### ISSUE TITLE
erratic behavior of the configuration of the requesting actors on targets

### ISSUE THREAD

**Issue by dgsn** [pluginsGLPI/formcreator#1648]
it's complicated.
See attached file : 
[acteur demandeur sur tickets.pdf](https://github.com/pluginsGLPI/formcreator/files/4138973/acteur.demandeur.sur.tickets.pdf)

Server
Windows Server 2012 R2 Standard Edition
Apache/2.4.16 (Win32) mod_authnz_sspi/0.1.0 OpenSSL/1.0.1p PHP/5.6.11 (Apache/2.4.16 (Win32) mod_authnz_sspi/0.1.0
OpenSSL/1.0.1p PHP/5.6.11 Server at
Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.117 Safari/537.36
Additional context
GLPI 9.4.5
formcreator Name: Form Creator Version: 2.9.1 State: Enabled

**Comment by btry** [comment]
Hi

From your attached  document

![image](https://user-images.githubusercontent.com/14139801/73542337-c7d78580-4434-11ea-899a-41a9cd75c388.png)

This actor has a problem : it is undefined.

It is very likely related to a bug I found this week, but the change is  in a bigger commit (code refactor).

Use a testing environment to test the following patch. After you applied the patch, delete the actor labelled N/A and create it again, then test the target is created as expected.

```diff
diff --git a/inc/targetbase.class.php b/inc/targetbase.class.php
index 53eb666f..9b888d8e 100644
--- a/inc/targetbase.class.php
+++ b/inc/targetbase.class.php
@@ -1097,7 +1097,7 @@ SCRIPT;
             'fieldtype' => ['glpiselect'],
             'values' => Group::class,
          ],
-         'actor_value_question_person',
+         'actor_value_question_' . PluginFormcreatorTarget_Actor::ACTOR_TYPE_QUESTION_GROUP,
          [
             'value' => 0
          ]
```

**Comment by dgsn** [comment]
now it is correct with the patch.

**Comment by btry** [comment]
Nice. The patch is included in the future version 2.10.

Thank you for your feedback.
